vllm/model_executor/models/mllama.py

```diff
@@ -795,19 +803,17 @@ def attention_with_mask(
         kv_len = k.shape[0]
         q = q.transpose(0, 1).view(self.num_local_key_value_heads,
                                    self.num_key_value_groups, q_len,
-                                   self.head_dim).contiguous()
+                                   self.head_dim)
         k = k.transpose(0,
                         1)[:,
                            None, :, :].expand(self.num_local_key_value_heads,
                                               self.num_key_value_groups,
-                                              kv_len,
-                                              self.head_dim).contiguous()
+                                              kv_len, self.head_dim)
         v = v.transpose(0,
                         1)[:,
                            None, :, :].expand(self.num_local_key_value_heads,
                                               self.num_key_value_groups,
-                                              kv_len,
-                                              self.head_dim).contiguous()
+                                              kv_len, self.head_dim)
```

Comment:
Why did you remove this? I added this intentionally to avoid illegal memory access in SDPA with batched multi-image

Reply:
Ooops, I didn't realize this when I merged from my another old fork to support BNB. :(

Have fixed this.